src/aliceVision/sfmData/colorize.cpp

```diff
@@ -81,13 +85,12 @@ void colorizeTracks(SfMData& sfmData)
             {
                 const Vec3& Tcenter = sfmData.getAbsolutePose(viewId).getTransform().center();
                 const Vec3& pt = landmark.X;
-                const double eucd = 1.0 / (Tcenter - pt).norm();
+                const double eucd = (Tcenter - pt).norm();
```

Comment:
If the observation is closer (smaller distance) we want to have a larger weighting, isn't it?

Reply:
Yes, thanks